rnns/fastrnns/bench.py

```diff
@@ -145,41 +153,44 @@ def bench(rnn_runners, group_name, print_json=False, sep=' ', **params):
     parser.add_argument('--print-json', action='store_true')
     parser.add_argument('--rnns', nargs='*',
                         help='What to run. cudnn, aten, jit, etc')
+    parser.add_argument('--cnns', nargs='*',
+                        help='What to run. resnet18, resnet18_jit, resnet50, etc')
 
     args = parser.parse_args()
     rnns = args.rnns or ['cudnn', 'aten', 'jit', 'jit_premul', 'jit_simple',
                          'jit_multilayer', 'py']
+    cnns = args.cnns or ['resnet18', 'resnet18_jit', 'resnet50', 'resnet50_jit']
     # TODO: Maybe add a separate section for the layernorm/dropout lstms
     # 'cudnn_layernorm', jit_layernorm', 'jit_layernom_decom',
     # 'jit', 'jit_dropout', 'cudnn_dropout'
     vlrnns = ['vl_cudnn', 'vl_jit', 'vl_py']
-    cnns = ['resnet18', 'resnet18_jit', 'resnet50', 'resnet50_jit']
     if args.print_json:
         print_stderr = lambda *args, **kwargs: None    # noqa
     print_stderr(args)
 
-    bench_args = vars(args)
+    bench_args = copy.deepcopy(vars(args))
     should_bench_varlen_lstms = args.variable_lstms
     del bench_args['rnns']
+    del bench_args['cnns']
     del bench_args['variable_lstms']
 
     if should_bench_varlen_lstms:
         if args.nloops + args.warmup > 30:
             print_stderr(
                 'WARNING: some of the variable sequence length lstms are '
                 'very unoptimized and therefore take forever to run.')
-        print_stderr('Benchmarking variable-length sequence LSTMs...')
-        rnn_results = bench(get_rnn_runners(*vlrnns), 'vl_lstm', **bench_args)
-        print_stderr('')
+        results = bench_group(vlrnns, 'variable-length sequence LSTM', 'vl_lstm', bench_args)
 
-    print_stderr('Benchmarking LSTMs...')
-    rnn_results = bench(get_rnn_runners(*rnns), 'lstm', **bench_args)
-    print_stderr('')
+    run_both = (args.cnns and args.rnns) or (not args.cnns and not args.rnns)
 
-    print_stderr('Benchmarking ResNets...')
-    cnn_results = bench(get_rnn_runners(*cnns), 'resnet', **bench_args)
-    print_stderr('')
+    if run_both:
```

Comment:
`run_both` seems redundant and won't scale well if we have more than two groups. Can't we do something like the following:
```
results = ... # Whatever the empty state is, I don't remember what this data structure is
if should_bench_varlen_lstms:
    results.update(bench_group(vlrnns, 'variable-length sequence LSTM', 'vl_lstm', bench_args))
if args.rnns:
    results.update(bench_group(rnns, 'LSTM', 'lstm', bench_args))
if args.cnns:
    results.update(bench_group(rnns, 'LSTM', 'lstm', bench_args))
```

Reply:
this is surely something we can do, the only thing for this is, if we don't pass any args to specify which group, we should run all groups of the benchmarks, which means we need to have something to know if we have `not arg.cnns and not arg.rnns` to be True

Reply:
Hmm one way to do this is to set default arguments for args.cnns and args.rnns via ArgParser

according to https://docs.python.org/3/library/argparse.html something like the following should work:
```
parser.add_argument('bar', nargs='*', default=[1, 2, 3], help='BAR!')
```

Reply:
Nice that should work 👍